src/assets/components/References.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+  <section id="references">       
+    <div class="text-content">
+      <h2>{{ text.title }}</h2>
+      <div>
+        <div
+          v-for="reference in text.references"
+          :key="reference.subTitle"
+        >
+          <p>
+            <span v-html="reference.num" />. <span v-html="reference.authors" /> <a
+              :href="reference.link"
+              target="_blank"
+            ><span v-html="reference.title" /></a><span v-html="reference.ref" />
+          </p>
+        </div>
+      </div>
+    </div>
+  </section>
+</template>
+  
+  <script>
+      import referencesText from "@/assets/text/referencesText";
+      export default {
+          name: 'References',
+          data() {
+              return {
+                  text: referencesText.referencesContent
+              }
+          }
+      }
+  </script>
+  /*Scope USWDS styles*/
```

Comment:
Suggest increasing the spacing after subtitles a little bit. If you're using name citations, eg Hammond et al. 2022,  should they also be numbered? I found that confusing.  I'd expect either alphabetical order in the references or numbers used in  the citation that corresponds to the order they appear

<img width="1281" alt="Screenshot 2023-07-21 at 7 43 24 AM" src="https://github.com/DOI-USGS/drought-timeline/assets/17803537/7c66e02c-1808-46b9-9f2a-036626520428">

Reply:
This format with the numbered list is the same one that we used on the what-is-drought site. I'll leave it for now for consistency, but if we want to change the format of the references for both, we could do that in the future.

I did also increase the spacing after sub-titles.

Reply:
But here it's more confusing because there's 3 subsections of references. Change this before merging